### ISSUE TITLE
After updating VS and .net 9 I have a problem with kits. All dependencies between projects are not working. The soluition doesn't want to compile. Is there a new parameter?

### ISSUE THREAD

**Issue by Alerinos** [dotnet/core#9183]
After updating VS and .net 9 I have a problem with kits. All dependencies between projects are not working. The soluition doesn't want to compile. Is there a new parameter?

_Originally posted by @Alerinos in https://github.com/dotnet/core/discussions/9167#discussioncomment-8460437_

**Comment by Alerinos** [comment]
Rolling back the version from .net 9 to .net 8 fixed the problem with project dependencies. Undoing the VS installation fixed the problem with packages.

**Comment by richlander** [comment]
Can you document the .NET and VS versions you had with the problem and w/o? That'd be helpful.

It would also be good if you could share any msbuild logs / errors.

**Comment by Alerinos** [comment]
@richlander 
Forgive me for not writing this earlier.
I am currently using VS 17.9.0 Preview 5.0 and .net 8.0.200-preview.23624.5 where everything is correct.
The problem occurs on VS 17.10.0 Preview 1.0 and .net 9 preview 1
Unfortunately I don't have the logs anymore because I restored the version.
The only error I have logged is NU1008, I am using PackageReference and Directory.Packages.props correctly
